api/rds.proto

```diff
@@ -285,12 +285,20 @@ message RouteAction {
 }
 
 message RedirectAction {
-  // A 302 redirect response will be sent which swaps the host portion of the
-  // URL with this value.
+  // The host portion of the URL will be swapped with this value.
   string host_redirect = 1;
-  // A 302 redirect response will be sent which swaps the path portion of the
-  // URL with this value.
+  // The path portion of the URL will be swapped with this value.
   string path_redirect = 2;
+
+  enum HTTPResponseCode {
+    // Moved Permanently HTTP Status Code - 301.
+    MOVED_PERMANENTLY = 0;
+    // Found HTTP Status Code - 302.
+    FOUND = 1;
```

Comment:
May be add 304 as well? From this (https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#3xx_Redirection), 304 seems to be the one that might be useful in future.

Reply:
Added 303 and 304 to keep the Status codes in order.

Reply:
Added 303 and 304 to keep the order.

Reply:
`304 Not Modified` definitely shouldn't be one this list, could you please remove it?

On the other hand, `307 Temporary Redirect` and `308 Permanent Redirect` are definitely worth adding.

Reply:
Yes good catch, I missed this. @ccaraman please fix.